config/load.go

```diff
@@ -17,22 +17,27 @@ func (c *Config) load() {
 	c.hotReloadableConfig = make(map[string][]*configValue)
 	c.envs = make(map[string]string)
 
-	if err := godotenv.Load(); err != nil && !isTest() {
-		fmt.Println("INFO: No .env file found.")
+	configSetupFromFiles := getEnv("CONFIG_SETUP_FROM_FILES", "true") == "true"
```

Comment:
IMO, this would require major modification in the initialisation code of the library, taking also into consideration the default `Config` instance that gets created during the package's `init()`

Reply:
This is to be configured by apps' `main` themselves, not customers.

Reply:
You have a point though, will revisit

Reply:
Just muting config error logs by default now and providing a means for rudder-server's container to enable them